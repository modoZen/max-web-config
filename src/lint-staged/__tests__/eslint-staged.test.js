const { getLintStagedConfig } = require('../eslint-staged');

describe('Lint Staged tests', () => {
  it('Should return lint staged config values', () => {
    expect(getLintStagedConfig()).toMatchSnapshot();
  });
});
