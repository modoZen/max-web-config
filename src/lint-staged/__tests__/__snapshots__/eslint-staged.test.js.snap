// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Lint Staged tests Should return lint staged config values 1`] = `
{
  "src/**/*.+(css|scss|sass)": [
    "prettier --write",
  ],
  "src/**/*.{js,jsx,ts,tsx}": [
    "eslint --fix",
  ],
}
`;
